### PR TITLE
[SQL] Fix recorded SQL tests for SDK version 0.26.0

### PR DESCRIPTION
**Pull request by Oleza1972**
Description

An engine change caused the below tests to break. This PR aims to fix that.
The fix adjusts recorded YAML files to unblock developers.

test_sql_db_import_export_mgmt.yaml
test_sql_failover_group_mgmt.yaml
test_sql_instance_pool.yaml
test_sql_managed_db_mgmt.yaml
test_sql_managed_db_short_retention.yaml
test_sql_managed_deleted_db_restore.yaml
test_sql_managed_instance_create_identity_mgmt.yaml
test_sql_managed_instance_mgmt.yaml
test_sql_mi_failover_mgmt.yaml
test_sql_midb_logreplay_mgmt.yaml
test_sql_virtual_cluster_mgmt.yaml
test_sql_db_sensitivity_classifications.yaml

There is also a fix in SqlDbSensitivityClassificationsScenarioTest class

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py [hunk 1]
```diff
@@ -4986,16 +4986,15 @@ def test_sql_db_sensitivity_classifications(self, resource_group, resource_group
...
-                     JMESPathCheck('informationTypeId', information_type_id),
-                     JMESPathCheck('labelId', label_id)])
+        response = self.cmd('sql db classification update -g {} -s {} -n {} --schema {} --table {} --column {} --information-type {} --label "{}"'
+                            .format(resource_group, server, database_name, schema_name, table_name, column_name, information_type, label_name),
+                            checks=[
+                                JMESPathCheck('informationType', information_type),
+                                JMESPathCheck('labelName', label_name)]).get_output_in_json()
+
+        information_type_id = response['informationTypeId']
+        label_id = response['labelId']
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:4997]: why not check them anymore?

**Comment by yonzhan** [comment]
SQL

**Comment by bradrich-msft** [comment]
Thank you for fixing test_sql_db_read_replica_mgmt.  readReplicaCount has been replaced by highAvailabilityReplicaCount.  If you don't mind, can you update that?

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py [hunk 2]
```diff
@@ -387,7 +387,7 @@ def test_sql_db_mgmt(self, resource_group, resource_group_location, server):
                            JMESPathCheck('status', 'Online'),
                            JMESPathCheck('zoneRedundant', False),
                            JMESPathCheck('readScale', 'Disabled'),
-                           JMESPathCheck('readReplicaCount', '0'),
```
Comment (bradrich-msft) [inline src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:390]: This is now 'highAvailabilityReplicaCount'

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py (hunk 1 above)
Comment (Oleza1972) [reply src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:4997]: They do checked at lines: 5005 and 5006.
They just can't hold pre-generated anymore.

**Comment by Oleza1972** [comment]
> Thank you for fixing test_sql_db_read_replica_mgmt. readReplicaCount has been replaced by highAvailabilityReplicaCount. If you don't mind, can you update that?

Done

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py [hunk 3]
```diff
@@ -387,7 +387,7 @@ def test_sql_db_mgmt(self, resource_group, resource_group_location, server):
...
+                           JMESPathCheck('highAvailabilityReplicaCount', None),
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py:390]: Please keep previous check value.

**Comment by Oleza1972** [comment]
Was created temporary to resolve test failures

**Pull request closed without merging**